### PR TITLE
Feature/worker node

### PR DESCRIPTION
**Pull request by pengin7384**
image
Done
- Implement worker node (REST API)

There are two job types. 'SEND' & 'VERIFY'.

### CODE REVIEW

**Comment by platfowner** [comment]
Hi, @seunghokimj, Could you review this PR?

**Comment by pengin7384** [comment]
Updated! PTAL @seunghokimj 

Test
```
POST http://localhost:3000/job
body:
{
  "type": "SEND",
  "config": {
    "time": 60,
    "number": 600,
    "ainUrl": "http://staging-node.ainetwork.ai:8080/",
    "ainAddress": "~~~",
    "ainPrivateKey": "~~~",
    "baseTx": {
      "operation": {
        "type": "SET_VALUE",
        "ref": "/apps/test/1",
        "value": 1
      },
    	"nonce": -1,
    	"address": "~~~"
    }
  }
}
```

File: package.json [hunk 1]
```diff
@@ -17,8 +17,11 @@
   },
```
Comment (seunghokimj) [inline package.json:17]: Add script for running worker node like `"worker-start": "node worker_node/index.js"`

Comment (pengin7384) [reply package.json:18]: Done! Add `"start-worker": "node worker_nodex/index.js"` & Update readme

File: worker_node/job/base.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+
+class Base {
+  #config;
+  #output;
+
+  constructor(config) {
+    this.#config = config;
```
Comment (seunghokimj) [inline worker_node/job/base.js:7]: Input "config" requires validation.
You also need to know what values will be in "config".

File: worker_node/job/send.js [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+    await delay(2 * BLOCK_TIME);
+
+    const writePermission = await this.#ain.db.ref(path).evalRule({value: null});
+    if (!writePermission) {
+      throw Error(`Can't write database (permission)`);
+    }
+  }
+
+  async sendTxs() {
+    const delayTime = this.config.time / this.config.number * 1000;
```
Comment (seunghokimj) [inline worker_node/job/send.js:86]: The meaning of variable names "number", "time" is ambiguous.
Suggests variable names such as "numberOfTransactions" and "duration".

File: worker_node/job/base.js (hunk 2 above)
Comment (pengin7384) [reply worker_node/job/base.js:7]: Done. Add validation logic to base

File: worker_node/job/send.js (hunk 3 above)
Comment (pengin7384) [reply worker_node/job/send.js]: Looks better. Done!

**Comment by pengin7384** [comment]
PTAL @seunghokimj

**Review by seunghokimj — APPROVED** [review]

**Pull request merged**